owncloud/owncloud.py

```diff
@@ -652,25 +652,41 @@ def copy(self, remote_path_source, remote_path_target):
         """
         return self.__webdav_move_copy(remote_path_source,remote_path_target,"COPY")
 
-    def share_file_with_link(self, path):
+    def share_file_with_link(self, path, **kwargs):
         """Shares a remote file with link
 
         :param path: path to the remote file to share
+        :param perms (optional): permission of the shared object
+        defaults to read only (1)
+        :param public_upload (optional): allows users to upload files or folders
+        :param password (optional): sets a password
+        http://doc.owncloud.org/server/6.0/admin_manual/sharing_api/index.html
         :returns: instance of :class:`PublicShare` with the share info
             or False if the operation failed
-        :raises: HTTPResponseError in case an HTTP error status was returned
+        :raises: ResponseError in case an HTTP error status was returned
```

Comment:
Did you forget to commit/push maybe ?
You can compare your local `git log` with the commit list from this PR